scripts/test-react.js

```diff
@@ -707,7 +707,9 @@ function runTestSuite(outputJsx, shouldTranspileSource) {
       });
 
       it("Class component as root with refs", async () => {
-        await runTest(directory, "class-root-with-refs.js");
+        await expectReconcilerFatalError(async () => {
+          await runTest(directory, "class-root-with-refs.js");
+        });
```

Comment:
I don't think it makes sense to mark this test as expecting a failure.

**The only reason we mutate `instance` there is to read it during the test.** It's not meant to be a test of "writing variables". It's a test of inlining things that use refs.

So if we can't use it approach, we just need to change **how the test reads it** — for example, by using `renderer.getInstance()` instead of a variable called `instance`.

We shouldn't expect this test to be broken, we should fix the test.

Reply:
Fixed the test.